.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-11, macos-latest, ubuntu-latest, windows-latest]
+        os: [macos-12, macos-13, macos-latest, ubuntu-latest, windows-latest]
```

Comment:
One x86 runner will suffice. If we get the 13 working, let's use it.

macos-12 reaches end of life before the end of the year and the binary compiled by the more modern runner should still be compatible.

Reply:
🔧 ✅